docs/components/DataList/Web.stories.tsx

```diff
@@ -95,6 +104,14 @@ const Template: ComponentStory<typeof DataList> = args => {
         created: "Created",
       }}
       onLoadMore={nextPage}
+      sorting={{
+        state: sortingState,
+        onSort: sorting => {
+          console.log(sorting);
+          setSortingState(sorting);
+        },
```

Comment:
do you still need the `console.log`? The setSortingState kinda tells you that it is working no?

Reply:
Yes, but it's a way of showing the results, as we do for `onSearch`